src/WebsocketClient/websocketclient.yml

```
@@ -16,4 +16,6 @@
       serverAddress: localhost
       serverPort: 5000
       collectLatency: true
-    arguments: "-c {{connections}} -w {{warmup}} -d {{duration}} -s {{scenario}} -u {% if serverUri == blank or serverUri == empty %}{{serverScheme}}://{{serverAddress}}:{{serverPort}}{{path}}{% else %}{{serverUri}}:{{serverPort}}{{path}}{% endif %}{% if collectLatency != blank %} --latency {{ collectLatency }}{% endif %}"
+      compress: false
+      size: 1000
+    arguments: "-c {{connections}} -w {{warmup}} -d {{duration}} -s {{scenario}} -u {% if serverUri == blank or serverUri == empty %}{{serverScheme}}://{{serverAddress}}:{{serverPort}}{{path}}{% else %}{{serverUri}}:{{serverPort}}{{path}}{% endif %}{% if collectLatency != blank %} --latency {{ collectLatency }}{% endif %} --size {{size}} --compress {{compress}}"
```

```suggestion
    arguments: "-c {{connections}} -w {{warmup}} -d {{duration}} -s {{scenario}} -u {{serverScheme}}://{{serverAddress}}:{{serverPort}}{{path}}{% if collectLatency != blank %} --latency {{ collectLatency }}{% endif %} --size {{size}} --compress {{compress}}"
```

Just removing some old arguments check that is not used anymore